src/main/java/com/fcgl/madrid/forum/model/Post.java

```diff
@@ -22,6 +22,7 @@
     private int likes;
```

Comment:
Shouldn't there be get and set methods for updatedDate, createdDate, likes, dislikes and cityID?

Reply:
Yes!!!! :)

Reply:
oh sorry, didn't see it. the file viewer didn't expand everything